### DISCUSSION TITLE
do commands not work on linux with the wayland window system?

### DISCUSSION THREAD

**Discussion by zany130** [hluk/CopyQ#2172] [Q&A]
Can't get any command to work tried several from the wiki such as https://github.com/hluk/CopyQ/wiki/Command-Examples#show-window-title

and I have this command to ignore bitwarden

```
[Command]
Automatic=true
Command="
    copyq ignore"
Icon=\xf069
Name=Ignore *\"Password\"* window
Remove=true
Window=Bitwarden
```
which is also not working

**Comment by hluk** [comment]
Unfortunately, apart from other things, getting/checking **window titles** is not supported on Wayland.

**Reply by zany130** [reply]
Ah i later thought that might be what's happening as other commands do work. So there no way for me to ignore the stuff copied from a specific app since we can't get the window title

**Reply by gabyx** [reply]
Oh soo sad... any hope for this?
